src/ui/popup/miscellaneous-settings/Sync-settings.tsx

```diff
@@ -0,0 +1,20 @@
+import {m} from 'malevic';
+import {ViewProps} from '../types';
+import CheckButton from '../check-button';
+
+export default function SyncSettings(props: ViewProps) {
+    function onSyncSettingsChange(checked: boolean) {
+        props.actions.changeSettings({syncSettings: checked});
+    }
+
+    return (
+        <CheckButton
+            checked={props.data.settings.syncSettings}
+            label="Enable syncing settings"
+            description={props.data.settings.syncSettings ?
+                'Enabled to sync settings' :
```

Comment:
Change to "Synchronized across devices"/"Not synchronized across devices" (if it's not too long).